docs/csharp/reference-semantics-with-value-types.md

```diff
@@ -35,34 +35,34 @@ edit the *csproj* file and add the following node:
 
 You can use either "7.2" or "latest" for the value.
 
-## Specifying `in` parameters
+## Passing arguments by readonly reference
 
 C# 7.2 adds the `in` keyword to complement the existing `ref`
-and `out` keywords when you write a method that passes arguments
-by reference. The `in` keyword specifies that you are passing
+and `out` keywords to pass arguments
+by reference. The `in` keyword specifies passing
 the parameter by reference and the called method does not modify
```

Comment:
parameter by reference --> argument by reference
comma before "the called method"
"but" seems better than "and" here